File: tundra-cli/benchmarks/recursive-fibonacci/fib.cpp
// Recursive Fibonacci in C++
#include <iostream>
#include <cstdlib>

long fib(long n) {
    if (n < 2) return n;
    return fib(n - 1) + fib(n - 2);
}

int main(int argc, char** argv) {
    long n = 30;
    std::cout << fib(n) << std::endl;
    return 0;
}

File: tundra-cli/benchmarks/recursive-fibonacci/fib.go
// Recursive Fibonacci in Go

package main

import (
    "fmt"
)

func fib(n int) int {
    if n < 2 {
        return n
    }
    return fib(n-1) + fib(n-2)
}

func main() {
    n := 30
    fmt.Println(fib(n))
}
